src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,22 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+/**
+ * Sort addressbook alphabetically
+ */
+public class SortCommand extends Command {
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": sort all persons alphabetically"
+            + "Example: " + COMMAND_WORD;
+
+    @Override
+    public CommandResult execute() {
+        addressBook.sort();
+        List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+        return new CommandResult("Sort finised", allPersons);
```

Comment:
It should be CommandResult("Sort finished", allPersons); =)
I think it would be consistent if you declare a CONSTANT to hold the value "Sort finished" since most of the Command Classes do that.